### DISCUSSION TITLE
Unable to sign up and start using Vercel - dashboard just hands with these errors

### DISCUSSION THREAD

**Discussion by goblinfactory** [vercel/community#3480] [Help] [answered]
### Summary

I'm a new developer, just finished a nextjs tutorial and at the end of the tutorial you're encouraged to create a vercel account and "deploy" your new app.

I've not used Vercel before. **I was impressed with the nextjs totorial so wanted to give Vercel a try and sign up**. 

1. I followed the signup process
2. I clicked "new project"  and then the screen (screenshot below) opened it just hangs with spinner in the "select a git namespace" dropdown. (see screenshot) . After waiting for more than a few minutes I pressed F12 and saw a host of errors, also visible in the screenshot and copy below; 

```logs
cookie.js:270 Refused to load the script 'https://m.servedby-buysellads.com/monetization.it.js' 
because it violates the following Content Security Policy directive: 
"script-src 'self' 'unsafe-eval' 'unsafe-inline' www.google.com www.googletagmanager.com snap.licdn.com 
googleads.g.doubleclick.net www.googleadservices.com www.google-analytics.com www.gstatic.com *.googleapis.com 
*.youtube.com *.youtube-nocookie.com *.ytimg.com *.twimg.com cdn.sift.com cdn.ampproject.org cdn.koala.live 
cdn.heapanalytics.com heapanalytics.com cdn.ethyca.com cdn.vercel-insights.com va.vercel-scripts.com vercel.com 
*.vercel.com *.vercel.sh vercel.live *.stripe.com twitter.com *.twitter.com *.github.com *.codesandbox.io https://*.googletagmanager.com:* wss://*.vercel.com localhost:* chrome-extension://*". 
Note that 'script-src-elem' was not explicitly set, so 'script-src' is used as a fallback.
```

Apart from the actual fact I can't sign up ...I'm quite concerned that there are so many badly managed scripts from ad-managers attempting to load random third party scripts onto a mission critical dashboard that has requested full unfettered access to my company github account.

This is quite worrying. Please advise! If I move beyond the free trial usage and become a paying customer, will you promise to remove all the bad random third party scripts? i.e. **do I have to PAY to make using the dashboard safe**?

Txs,

A
<img width="1578" alt="Screenshot 2023-08-02 at 18 05 40" src="https://github.com/vercel/community/assets/1073487/8743f39c-0396-4456-bf38-a828704e99e4">

### Steps to Reproduce

- Sign up as a new user using Chrome on mac
- Click "create new project"
- Dashboard hangs
- Press F12 to see any error messages

**Comment by swarnava** [comment]
Hey there,

This doesn't seem like a script injected by us. Are you also  able to reproduce this In Incognito without any browser extension?

**Reply by goblinfactory** [reply]
Hi Swarnava

I did what you said, disabled extensions and tried again using incognito, and got the same response (See screenshot below) 
The only extensions I had turned on (during the first attempt) was the official react extension and google offline docs extention, both of which I disabled for the second test as per your suggestion. I am security paranoid and don't install extensions.
 
Thank you
Alan

 
<img width="1402" alt="Screenshot 2023-08-02 at 21 09 25" src="https://github.com/vercel/community/assets/1073487/2e8e4bdf-2090-4cc9-9148-93ca15b648de">

**Comment by swarnava** [comment]
Hi Alan,

Do you have a different device where you can try this? I wonder if this is
some sort of browser malware which pushing the script.

**Reply by goblinfactory** [reply]
I just tried it again, on my clean un-used backup machine, (recently formatted, re-installed from factory mac, and not used) 
I did the same test, chrome, no extensions, and using incognito.
same hanging, and same primary error message in the console. (F12), "refused to load the ..."
I am not certain that they are related?
your marketing team might have a requirement in place that certain page progression is blocked until the tracking scripts are loaded. You'll need to confirm that with them.

depends on what the startup lifecycle is for the dashboard?
1. is the "select a GIT namespace" dropdown picker being blocked by the tracking scripts needing to be loaded first?

<img width="492" alt="Screenshot 2023-08-02 at 21 47 29" src="https://github.com/vercel/community/assets/1073487/cfadb619-d754-4f4d-9a39-53c8e786b61a">

if so, then yes, that's the reason the page is hanging.
If NO, then we may simply be seeing two problems, the hanging could merely be what's making us think the tracking scripts is related. 

It's quite possible the tracking scripts behave this way for most users, but most users dont press F12 at exactly this screen because their GIT namespace picker does not hang and they move on and no one checks the console log for errors or problems.

i.e. it's only when you navigate to the route : https://vercel.com/new that this problem surfaces. 

what is interesting is that if you don't press the "add new project" button, and remain at the parent route path, you will see a perfectly clean console with the "I'm guessing" familiar Vercel welcome; 

<img width="438" alt="Screenshot 2023-08-02 at 21 52 24" src="https://github.com/vercel/community/assets/1073487/0b4403f3-c785-4aa9-9ffc-5ca9da28e8c1">

This is at route : vercel.com/dashboard ( no click of "new project button" yet.)

**Comment by goblinfactory** [comment]
@swarnava
 I'm going to delete my account and create a new personal account. I suspect we have two problems;
 Possibly the first is related to the GIT namespace picker. If that's screwed up somehow that would explain the hanging.
 
 let's turn this support request into a single requests for assistance, focusing only on the hanging.
 I'll open a new support ticket after we've got me logging in. I know lots of people I trust that use Vercel so I expect we should be able to get past this. I'm pretty certain a delete and re-create account will sort this. (you don't have a facility to remote and re-connect github, so I'll just close and sign up again. shouldnt take long)
 
 Once that's sorted, I'll log a separate request to ask about the tracking scripts, taht's a concerns for sure, but it's a different concern and I can still test Vercel in the meantime.
 give me a few minutes to delete my test Vercel account  and create a new one.
A

**Comment by goblinfactory** [comment, marked as answer]
Hi @swarnava 
I'm now finally in, deleted and re-created account and re-attached it to github, ... let me do some work, then I'll come back and provide some additional information/ context.

**Comment by goblinfactory** [comment]
I'm going to mark my comment above as answer, while it can be improved and there's some valuable additional information that I can share, since I believe I'm not the only person that this issue will impact. ... I have some important work I need to do first. ciao, chat later.A

**Comment by swarnava** [comment]
Thank you! I am glad to hear this is fixed now
